userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,124 @@
+
+Given an image and a one-shot detector trained on category **stop sign**, the output prediction will look like:
+
+![Stop Sign vector and prediction images](images/stop_sign_sample.jpg) 
+
+
+#### Introductory Example
+
+In this example, the goal is to **localize** where in the image instances of the category **stop sign** are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```python
+import turicreate as tc
+
+# Load the starter images
+starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
+                   'label':['stop_sign']})
+
+# Load test images
+test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'), 
+                                tc.Image('stop_sign_photo2.png')],
+                        'label':['stop_sign', 'stop_sign']})
+
+# Create a model. This step may take several hours                                      
+model = tc.one_shot_object_detector.create(starter_images, 'label')
+
+# Save predictions on the test set as a column of the test data SFrame
+test_data['predictions'] = model.predict(test_data)
+
+# Draw prediction bounding boxes on the test images
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions']) 
+
+# Evaluate the model and save the results in a dictionary
+metrics = model.evaluate(test_data)
+
+# Save the model for later use in TuriCreate
+model.save('mymodel.model')
+
+# Export for use in Core ML
+model.export_coreml('MyCustomOneShotDetector.mlmodel')
+```
+
+Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
+
+
+```python
+augmented_images = \
+	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
+
+augmented_images.explore()
+```
+
+![Explore augmented images](images/augmented_images_explore.jpg)
+
+Some examples of synthetic training data generated:
+
+![Sample augmented images](images/augmented_images_collage.jpg)
+
+*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
+
+
+To visualize the predictions made on the test set:
+
+
+```python
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
+
+test_data.explore()
+```
+
+![Prediction test data](images/test_data_explore.jpg)
+
+Examples of test image predictions:
+
+![Sample prediction image - 1 and 2](images/sample_prediction_images_1_2.jpg)
+
+
+#### How it works
```

Comment:
I think you are missing an intro paragraph here to set up how this works. 

You want to say something to the effect of - This works in 2 steps, the first step is data generation and the second step is training an OD model. Then you want to explain the data generation step in a bit more details and then the model creation in detail. That would put the whole thing into a lot more context.

Reply:
I feel like "generation" introduces a 3rd word (in addition to "augmentation" and "synthesis"). Should we call this "data synthesis" here as well, for consistency?